### DISCUSSION TITLE
[Feature] Import from JetLog

### DISCUSSION THREAD

**Discussion by edersong** [johanohly/AirTrail#23] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I deployed JetLog https://github.com/pbogre/jetlog and put all of my flight's history ther, but now found AirTrail that is so much better, so would like a way to import my data from JetLog.

JetLog exported csv format is:
`date,origin,destination,departure_time,arrival_time,arrival_date,seat,ticket_class,duration,distance,airplane,flight_number,notes`

**Comment by johanohly** [comment]
This will be available in the next release (v0.1.2)
